gcloud-java-compute/src/main/java/com/google/gcloud/compute/Compute.java

```diff
@@ -1525,4 +1640,53 @@ public static AddressAggregatedListOption pageToken(String pageToken) {
    * @throws ComputeException upon failure
    */
   Operation delete(AddressId addressId, OperationOption... options);
+
+  /**
+   * Creates a new snapshot.
+   *
+   * @return a zone operation if the create request was issued correctly, {@code null} if
+   *     {@code snapshot.sourceDisk} was not found
+   * @throws ComputeException upon failure
+   */
+  Operation create(SnapshotInfo snapshot, OperationOption... options);
+
+  /**
+   * Returns the requested snapshot or {@code null} if not found.
+   *
+   * @throws ComputeException upon failure
+   */
+  Snapshot getSnapshot(String snapshot, SnapshotOption... options);
+
+  /**
+   * Lists all snapshots.
```

Comment:
That's valid.  It also looks like the [list is capped at 500 per call]( https://cloud.google.com/compute/docs/reference/latest/snapshots/list#maxResults) though.  I imagine this won't be an issue for most people, but it seems like a minor doc change to be slightly more correct.